cypress/integration/home.spec.js

```diff
@@ -0,0 +1,69 @@
+const headerItemText = [
+  "API CATALOGUE",
+  "SIGN IN",
+];
+
+const sidebarItemText = [
+    "Mission",
+    "The need of a developer hub",
+    "API Specifications",
+    "Our ways of working",
+];
+
+const footerItemText = [
+  "Hackney sites",
+  "Accessibility",
+  "Contact us",
+];
+
+function elementContainsTextArray(element, textArray) {
+  element.each((item, index) => {
+    expect(item.text()).to.eq(textArray[index]);
+  });
+}
+
+describe("Home page integration test", () => {
+  beforeEach(() => {
+    cy.visit("/")
+  });
+
+  it("Navbar test", () => {
+    const header = cy.get("#header");
+
+    // find logo
+    cy.get(".lbh-header__logo-fallback-image");
+
+    // find title
+    const headerTitle = header.get(".name");
+    headerTitle.contains("DEVELOPER HUB");
+
+    // find header items
+    const headerItems = header.get(".nav-items").children();
+    elementContainsTextArray(
+      headerItems,
+      headerItemText
+    );
+  });
+
+  it("Sidebar test", () => {
+    const sidebar = cy.get(".sidebar");
```

Comment:
Maybe test on multiple screen sizes (look at some of the other test files)

Reply:
when i get some more time I will try to do it, however if someone has more time to test on multiple screen sizes is more than welcomed

Reply:
By that I mean implementing automated checks on various screen sizes. To do this, you can import the screen sizes like this:
```
import { screenSizes } from "../support/screenSizes";
```
then put the tests in a for loop like this:
```
screenSizes.forEach((screenSize) => {
   // tests here
}
```

See [this test file](https://github.com/LBHackney-IT/developers-hub-frontend/blob/master/cypress/integration/previewApi.spec.js) for an example